### ISSUE TITLE
Blanks filter crashes Excel

### ISSUE THREAD

**Issue by harri-rajala-koivu-solutions** [OfficeDev/office-js#465]
Using office-js Excel API, Excel crashes and restarts when (Blanks) filter is selected and you try to load column filter.

## Expected Behavior
Excel should return column filter and not crash.

## Current Behavior
Excel crashes and restarts.

## Steps to Reproduce, or Live Example

Summary of reproduce steps:

Step 1: Excel JS code. Create table with two columns.
Step 2: Manual step. Set filter to use values "b2", "b3", "b4" and (Blanks).
Step 3: Excel JS code. Excel crashes when we try to read column filter.

Detailed steps:

### Step 1

Execute following code:

```typescript
    private excelCrashStep1 = async () => {
        await Excel.run(async (context: Excel.RequestContext) => {
            const sheet = context.workbook.worksheets.getActiveWorksheet();
            const table = sheet.tables.add("A1:B1", true);
            table.name = "TestTable";
            table.getHeaderRowRange().values = [["Test1", "Filter Column"]];
            table.rows.add(undefined, [
                ["a1", "b1"],
                ["a2", "b2"],
                ["a3", "b3"],
                ["a3", "b4"],
                ["a4", ""]
            ]);
            await context.sync();
        });
    }
```

### Step 2
**!!! IMPORTANT NOTE !!!**
This must be done manually because we don't know how to apply (Blanks) filter using Excel JS.

* Open filter menu from second column (Filter Column)
* You should see following values:
```
(Select All)
b1
b2
b3
b4
(Blanks)
```
* Now **unselect "b1"**
* You should have these 3 values as a selected filter
```
b2
b3
b4
(Blanks) 
```
* Click Ok
* Now you can go to Step 3

### Step 3

Execute following code:

```typescript
    private excelCrashStep3 = async () => {
        await Excel.run(async (context: Excel.RequestContext) => {
            const sheet = context.workbook.worksheets.getActiveWorksheet();
            const table = sheet.tables.getItem("TestTable");
            table.load("columns");
            await context.sync();

            const column = table.columns.items[1];
            column.load("filter");
            await context.sync(); // !!! Excel crashes here

            sheet.getRange("D1").values = [[JSON.stringify(column.filter)]];
            await context.sync();
        });
    }
```

## Context
This is one case where we have found Excel crashing with office-js API and are able to reliable repeat it. There seems to be many other cases, too, that we are still studying and trying to find code to repeat them.

Note that if you unselect also (Blanks) from filter in Step 2 then Excel does not crash. But in our case it is end user that can choose any filter combinations. After that our Excel add-in reads filter and Excel crashes.

## Your Environment
* Platform [PC desktop, Mac, iOS, Office Online]: PC Desktop
* Host [Excel, Word, PowerPoint, etc.]: Excel
* Office version number: 1902 build 11328.20222 32-bit
* Operating System: Windows 10 Home version 1803 (OS Build 17134.648)
* Browser (if using Office Online):

## Useful logs
- **Application Error log** in Event Viewer

```
Faulting application name: EXCEL.EXE, version: 16.0.11328.20222, time stamp: 0x5c95d1e1
Faulting module name: EXCEL.EXE, version: 16.0.11328.20222, time stamp: 0x5c95d1e1
Exception code: 0xc0000005
Fault offset: 0x008b69db
Faulting process id: 0x4b20
Faulting application start time: 0x01d4eba47b5c525f
Faulting application path: C:\Program Files (x86)\Microsoft Office\root\Office16\EXCEL.EXE
Faulting module path: C:\Program Files (x86)\Microsoft Office\root\Office16\EXCEL.EXE
Report Id: bdbf8fef-06ba-40b1-bd40-955d2f3576d3
Faulting package full name: 
Faulting package-relative application ID: 
```

- **Windows Error Reporting** in Event Viewer
```
Fault bucket 1928515946242973528, type 1
Event Name: APPCRASH
Response: Not available
Cab Id: 0

Problem signature:
P1: EXCEL.EXE
P2: 16.0.11328.20222
P3: 5c95d1e1
P4: EXCEL.EXE
P5: 16.0.11328.20222
P6: 5c95d1e1
P7: c0000005
P8: 008b69db
P9: 
P10: 

Attached files:
\\?\C:\Users\Harri\AppData\Local\Temp\{BB72D0A1-0773-4C01-8AA4-0F96E6BE9E40} - OProcSessId.dat
\\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WERFB3E.tmp.WERInternalMetadata.xml
\\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WERFB5E.tmp.xml
\\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WERFB6D.tmp.csv
\\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WERFB9C.tmp.txt

These files may be available here:
C:\ProgramData\Microsoft\Windows\WER\ReportArchive\AppCrash_EXCEL.EXE_8a6687ea5d98a927a9d61a791791becd78e5e2_00000000_7c17008d

Analysis symbol: 
Rechecking for solution: 0
Report Id: bdbf8fef-06ba-40b1-bd40-955d2f3576d3
Report Status: 268435456
Hashed bucket: c9e33881755b01ebfac377090f7d1f58
Cab Guid: 0
```

Content of **Report.wer** file in C:\ProgramData\Microsoft\Windows\WER\ReportArchive\AppCrash_EXCEL.EXE_8a6687ea5d98a927a9d61a791791becd78e5e2_00000000_7c17008d

```
Version=1
EventType=APPCRASH
EventTime=131989381102100392
ReportType=2
Consent=1
UploadTime=131989381103637010
ReportStatus=268435456
ReportIdentifier=d05dc3e2-7485-49a2-bc6b-b9f1ea41fbda
IntegratorReportIdentifier=bdbf8fef-06ba-40b1-bd40-955d2f3576d3
Wow64Host=34404
Wow64Guest=332
OriginalFilename=Excel.exe
AppSessionGuid=00004b20-0003-0034-5f52-5c7ba4ebd401
TargetAppId=W:0006171b160a18280e1fd3d283a4a4d72d1b00000000!000015b3fc229f08917df241d927aedd61d8b7dc5f52!EXCEL.EXE
TargetAppVer=2019//03//23:06:27:45!2973cfd!EXCEL.EXE
BootId=4294967295
TargetAsId=34567
IsFatal=4294967295
Response.BucketId=c9e33881755b01ebfac377090f7d1f58
Response.BucketTable=1
Response.LegacyBucketId=1928515946242973528
Response.type=4
Sig[0].Name=Application Name
Sig[0].Value=EXCEL.EXE
Sig[1].Name=Application Version
Sig[1].Value=16.0.11328.20222
Sig[2].Name=Application Timestamp
Sig[2].Value=5c95d1e1
Sig[3].Name=Fault Module Name
Sig[3].Value=EXCEL.EXE
Sig[4].Name=Fault Module Version
Sig[4].Value=16.0.11328.20222
Sig[5].Name=Fault Module Timestamp
Sig[5].Value=5c95d1e1
Sig[6].Name=Exception Code
Sig[6].Value=c0000005
Sig[7].Name=Exception Offset
Sig[7].Value=008b69db
DynamicSig[1].Name=OS Version
DynamicSig[1].Value=10.0.17134.2.0.0.768.101
DynamicSig[2].Name=Locale ID
DynamicSig[2].Value=8192
UI[2]=C:\Program Files (x86)\Microsoft Office\root\Office16\EXCEL.EXE
LoadedModule[0]=C:\Program Files (x86)\Microsoft Office\root\Office16\EXCEL.EXE
LoadedModule[1]=C:\WINDOWS\SYSTEM32\ntdll.dll
LoadedModule[2]=C:\WINDOWS\System32\KERNEL32.DLL
LoadedModule[3]=C:\WINDOWS\System32\KERNELBASE.dll
LoadedModule[4]=C:\WINDOWS\SYSTEM32\apphelp.dll
LoadedModule[5]=C:\WINDOWS\System32\ole32.dll
LoadedModule[6]=C:\WINDOWS\System32\combase.dll
LoadedModule[7]=C:\WINDOWS\System32\ucrtbase.dll
LoadedModule[8]=C:\Program Files (x86)\Microsoft Office\root\Office16\AppVIsvSubsystems32.dll
LoadedModule[9]=C:\WINDOWS\System32\RPCRT4.dll
LoadedModule[10]=C:\WINDOWS\System32\SspiCli.dll
LoadedModule[11]=C:\WINDOWS\System32\CRYPTBASE.dll
LoadedModule[12]=C:\WINDOWS\System32\bcryptPrimitives.dll
LoadedModule[13]=C:\WINDOWS\System32\sechost.dll
LoadedModule[14]=C:\WINDOWS\System32\GDI32.dll
LoadedModule[15]=C:\WINDOWS\System32\gdi32full.dll
LoadedModule[16]=C:\WINDOWS\System32\msvcp_win.dll
LoadedModule[17]=C:\WINDOWS\System32\USER32.dll
LoadedModule[18]=C:\WINDOWS\System32\win32u.dll
LoadedModule[19]=C:\Program Files (x86)\Microsoft Office\root\Office16\c2r32.dll
LoadedModule[20]=C:\WINDOWS\System32\ADVAPI32.dll
LoadedModule[21]=C:\WINDOWS\System32\OLEAUT32.dll
LoadedModule[22]=C:\WINDOWS\System32\msvcrt.dll
LoadedModule[23]=C:\WINDOWS\System32\SHELL32.dll
LoadedModule[24]=C:\WINDOWS\System32\cfgmgr32.dll
LoadedModule[25]=C:\WINDOWS\System32\shcore.dll
LoadedModule[26]=C:\WINDOWS\System32\windows.storage.dll
LoadedModule[27]=C:\WINDOWS\SYSTEM32\VCRUNTIME140.dll
LoadedModule[28]=C:\WINDOWS\SYSTEM32\MSVCP140.dll
LoadedModule[29]=C:\WINDOWS\System32\shlwapi.dll
LoadedModule[30]=C:\WINDOWS\System32\kernel.appcore.dll
LoadedModule[31]=C:\WINDOWS\System32\profapi.dll
LoadedModule[32]=C:\WINDOWS\System32\powrprof.dll
LoadedModule[33]=C:\WINDOWS\System32\FLTLIB.DLL
LoadedModule[34]=C:\WINDOWS\SYSTEM32\USERENV.dll
LoadedModule[35]=C:\WINDOWS\System32\IMM32.DLL
LoadedModule[36]=C:\Program Files (x86)\Common Files\Microsoft Shared\Office16\mso20win32client.dll
LoadedModule[37]=C:\Program Files (x86)\Common Files\Microsoft Shared\Office16\mso30win32client.dll
LoadedModule[38]=C:\Program Files (x86)\Common Files\Microsoft Shared\Office16\mso40uiwin32client.dll
LoadedModule[39]=C:\WINDOWS\WinSxS\x86_microsoft.windows.gdiplus_6595b64144ccf1df_1.1.17134.648_none_73ddcaa59f7149a6\gdiplus.dll
LoadedModule[40]=C:\WINDOWS\SYSTEM32\dwmapi.dll
LoadedModule[41]=C:\Program Files (x86)\Common Files\Microsoft Shared\Office16\mso50win32client.dll
LoadedModule[42]=C:\Program Files (x86)\Common Files\Microsoft Shared\Office16\mso98win32client.dll
LoadedModule[43]=C:\WINDOWS\SYSTEM32\WTSAPI32.dll
LoadedModule[44]=C:\Program Files (x86)\Common Files\Microsoft Shared\Office16\mso.dll
LoadedModule[45]=C:\WINDOWS\SYSTEM32\VERSION.dll
LoadedModule[46]=C:\WINDOWS\SYSTEM32\msi.dll
LoadedModule[47]=C:\WINDOWS\SYSTEM32\bcrypt.dll
LoadedModule[48]=C:\WINDOWS\WinSxS\x86_microsoft.windows.common-controls_6595b64144ccf1df_6.0.17134.648_none_42f2d7c044decfd8\Comctl32.dll
LoadedModule[49]=C:\WINDOWS\SYSTEM32\d2d1.dll
LoadedModule[50]=C:\WINDOWS\system32\uxtheme.dll
LoadedModule[51]=C:\WINDOWS\System32\MSCTF.dll
LoadedModule[52]=C:\WINDOWS\SYSTEM32\WINSTA.dll
LoadedModule[53]=C:\WINDOWS\SYSTEM32\dxgi.dll
LoadedModule[54]=C:\WINDOWS\SYSTEM32\d3d11.dll
LoadedModule[55]=C:\WINDOWS\System32\DriverStore\FileRepository\igdlh64.inf_amd64_f1d1d01ad984f535\igd10iumd32.dll
LoadedModule[56]=C:\WINDOWS\SYSTEM32\ncrypt.dll
LoadedModule[57]=C:\WINDOWS\SYSTEM32\NTASN1.dll
LoadedModule[58]=C:\WINDOWS\System32\DriverStore\FileRepository\igdlh64.inf_amd64_f1d1d01ad984f535\igc32.dll
LoadedModule[59]=C:\WINDOWS\SYSTEM32\ntmarta.dll
LoadedModule[60]=C:\WINDOWS\SYSTEM32\Secur32.dll
LoadedModule[61]=C:\WINDOWS\System32\clbcatq.dll
LoadedModule[62]=C:\Windows\System32\Windows.Security.Authentication.Web.Core.dll
LoadedModule[63]=C:\WINDOWS\System32\CRYPT32.dll
LoadedModule[64]=C:\WINDOWS\System32\MSASN1.dll
LoadedModule[65]=C:\Windows\System32\AppXDeploymentClient.dll
LoadedModule[66]=C:\WINDOWS\SYSTEM32\wintypes.dll
LoadedModule[67]=C:\Windows\System32\twinapi.appcore.dll
LoadedModule[68]=C:\Windows\System32\StateRepository.Core.dll
LoadedModule[69]=C:\Windows\System32\RMCLIENT.dll
LoadedModule[70]=C:\WINDOWS\SYSTEM32\mscoree.dll
LoadedModule[71]=C:\Windows\Microsoft.NET\Framework\v4.0.30319\mscoreei.dll
LoadedModule[72]=C:\WINDOWS\SYSTEM32\DWrite.dll
LoadedModule[73]=C:\Program Files (x86)\Common Files\Microsoft Shared\Office16\riched20.dll
LoadedModule[74]=C:\WINDOWS\SYSTEM32\XmlLite.dll
LoadedModule[75]=C:\WINDOWS\System32\netprofm.dll
LoadedModule[76]=C:\WINDOWS\System32\WS2_32.dll
LoadedModule[77]=C:\WINDOWS\System32\npmproxy.dll
LoadedModule[78]=C:\WINDOWS\SYSTEM32\IPHLPAPI.DLL
LoadedModule[79]=C:\WINDOWS\System32\NSI.dll
LoadedModule[80]=C:\WINDOWS\SYSTEM32\webservices.dll
LoadedModule[81]=C:\Windows\System32\Windows.System.Profile.RetailInfo.dll
LoadedModule[82]=C:\Windows\System32\iertutil.dll
LoadedModule[83]=C:\WINDOWS\SYSTEM32\sppc.dll
LoadedModule[84]=C:\Windows\System32\OneCoreCommonProxyStub.dll
LoadedModule[85]=C:\Windows\System32\vaultcli.dll
LoadedModule[86]=C:\Windows\System32\Windows.Web.dll
LoadedModule[87]=C:\WINDOWS\System32\DPAPI.dll
LoadedModule[88]=C:\WINDOWS\System32\Normaliz.dll
LoadedModule[89]=C:\WINDOWS\SYSTEM32\CRYPTSP.dll
LoadedModule[90]=C:\WINDOWS\SYSTEM32\WINHTTP.dll
LoadedModule[91]=C:\WINDOWS\system32\rsaenh.dll
LoadedModule[92]=C:\WINDOWS\SYSTEM32\dhcpcsvc6.DLL
LoadedModule[93]=C:\WINDOWS\SYSTEM32\dhcpcsvc.DLL
LoadedModule[94]=C:\Windows\System32\aadWamExtension.dll
LoadedModule[95]=C:\WINDOWS\SYSTEM32\urlmon.dll
LoadedModule[96]=C:\WINDOWS\SYSTEM32\WININET.dll
LoadedModule[97]=C:\WINDOWS\SYSTEM32\ondemandconnroutehelper.dll
LoadedModule[98]=C:\WINDOWS\system32\mswsock.dll
LoadedModule[99]=C:\WINDOWS\SYSTEM32\WINNSI.DLL
LoadedModule[100]=C:\WINDOWS\SYSTEM32\webio.dll
LoadedModule[101]=C:\WINDOWS\SYSTEM32\DNSAPI.dll
LoadedModule[102]=C:\Windows\System32\rasadhlp.dll
LoadedModule[103]=C:\WINDOWS\System32\fwpuclnt.dll
LoadedModule[104]=C:\WINDOWS\System32\schannel.dll
LoadedModule[105]=C:\Program Files (x86)\Microsoft Office\root\Office16\oart.dll
LoadedModule[106]=C:\WINDOWS\SYSTEM32\dcomp.dll
LoadedModule[107]=C:\WINDOWS\system32\twinapi.dll
LoadedModule[108]=C:\WINDOWS\system32\TextInputFramework.dll
LoadedModule[109]=C:\WINDOWS\system32\CoreUIComponents.dll
LoadedModule[110]=C:\WINDOWS\system32\CoreMessaging.dll
LoadedModule[111]=C:\Windows\System32\msxml6.dll
LoadedModule[112]=C:\WINDOWS\SYSTEM32\mskeyprotect.dll
LoadedModule[113]=C:\WINDOWS\system32\ncryptsslp.dll
LoadedModule[114]=C:\WINDOWS\SYSTEM32\d3d10_1.dll
LoadedModule[115]=C:\WINDOWS\SYSTEM32\d3d10_1core.dll
LoadedModule[116]=C:\WINDOWS\SYSTEM32\WindowsCodecs.dll
LoadedModule[117]=c:\Program Files (x86)\Common Files\Microsoft Shared\VSTO\vstoee.dll
LoadedModule[118]=c:\Program Files (x86)\Common Files\Microsoft Shared\VSTO\10.0\VSTOLoader.dll
LoadedModule[119]=C:\WINDOWS\SYSTEM32\MSVCR100.dll
LoadedModule[120]=C:\WINDOWS\SYSTEM32\MSVCP100.dll
LoadedModule[121]=C:\WINDOWS\SYSTEM32\oledlg.dll
LoadedModule[122]=C:\WINDOWS\system32\directmanipulation.dll
LoadedModule[123]=C:\WINDOWS\SYSTEM32\MSIMG32.dll
LoadedModule[124]=C:\WINDOWS\SYSTEM32\usp10.dll
LoadedModule[125]=C:\Windows\System32\Windows.Globalization.dll
LoadedModule[126]=C:\Windows\System32\Bcp47Langs.dll
LoadedModule[127]=C:\Windows\System32\bcp47mrm.dll
LoadedModule[128]=C:\WINDOWS\SYSTEM32\globinputhost.dll
LoadedModule[129]=C:\Program Files (x86)\Common Files\Microsoft Shared\Office16\MSPTLS.DLL
LoadedModule[130]=C:\WINDOWS\System32\msxml3.dll
LoadedModule[131]=C:\Windows\Microsoft.NET\Framework\v4.0.30319\clr.dll
LoadedModule[132]=C:\WINDOWS\SYSTEM32\MSVCR120_CLR0400.dll
LoadedModule[133]=C:\Windows\System32\cryptnet.dll
LoadedModule[134]=C:\WINDOWS\assembly\NativeImages_v4.0.30319_32\mscorlib\9e5fb54bb42672f30445fd7d36d2f815\mscorlib.ni.dll
LoadedModule[135]=C:\WINDOWS\assembly\NativeImages_v4.0.30319_32\System\c0994859a3042efc9742c38560611eaf\System.ni.dll
LoadedModule[136]=C:\WINDOWS\assembly\NativeImages_v4.0.30319_32\Microsoft.V5499e76b#\93f3d4e86e4bff84085175fe00dc761f\Microsoft.VisualStudio.Tools.Office.Runtime.Internal.ni.dll
LoadedModule[137]=C:\WINDOWS\assembly\NativeImages_v4.0.30319_32\Microsoft.O5949707a#\a22222fab8f47de52a828a5368bfaf06\Microsoft.Office.Tools.ni.dll
LoadedModule[138]=C:\WINDOWS\assembly\NativeImages_v4.0.30319_32\System.Core\4f172eb9ef81ba1bd6013111bb28d877\System.Core.ni.dll
LoadedModule[139]=C:\WINDOWS\assembly\NativeImages_v4.0.30319_32\Microsoft.V883708cb#\f8b0a8d460b43e38600d63e271f23cb4\Microsoft.VisualStudio.Tools.Applications.Runtime.ni.dll
LoadedModule[140]=C:\WINDOWS\assembly\NativeImages_v4.0.30319_32\Microsoft.Vf9a08577#\fe911a10b6d5f83c7938d1efd2f9ca03\Microsoft.VisualStudio.Tools.Applications.Hosting.ni.dll
LoadedModule[141]=C:\WINDOWS\system32\mlang.dll
LoadedModule[142]=C:\WINDOWS\assembly\NativeImages_v4.0.30319_32\Microsoft.Oe2e369ff#\e6228106873186d522570e010c939560\Microsoft.Office.Tools.Excel.Implementation.ni.dll
LoadedModule[143]=C:\WINDOWS\assembly\NativeImages_v4.0.30319_32\Microsoft.O4a946565#\eda2ce1daf82b162a8881c4893d43755\Microsoft.Office.Tools.Common.Implementation.ni.dll
LoadedModule[144]=C:\WINDOWS\assembly\NativeImages_v4.0.30319_32\Microsoft.O854200f9#\fd4a3ca6a5bf12e47584dd2037be7fd9\Microsoft.Office.Tools.Common.ni.dll
LoadedModule[145]=C:\WINDOWS\assembly\NativeImages_v4.0.30319_32\Microsoft.O4161a3a3#\8ebd0d9b4e6f6becfcf9c737c0d96ec6\Microsoft.Office.Tools.Excel.ni.dll
LoadedModule[146]=C:\WINDOWS\assembly\NativeImages_v4.0.30319_32\Microsoft.O2eb0cc9a#\83237a09f5fa65c83dffae5756f31771\Microsoft.Office.Tools.v4.0.Framework.ni.dll
LoadedModule[147]=C:\WINDOWS\assembly\NativeImages_v4.0.30319_32\System.Drawing\ea30ae1302ede2b30d7b70fe65cf12d8\System.Drawing.ni.dll
LoadedModule[148]=C:\WINDOWS\assembly\NativeImages_v4.0.30319_32\System.Windows.Forms\63fd619b39b7d0d303607ce0f7132db8\System.Windows.Forms.ni.dll
LoadedModule[149]=C:\Windows\Microsoft.NET\Framework\v4.0.30319\clrjit.dll
LoadedModule[150]=C:\Program Files\Common Files\Microsoft Shared\Team Foundation Server\15.0\x86\TFSOfficeAdd-in.dll
LoadedModule[151]=C:\WINDOWS\SYSTEM32\sxs.dll
LoadedModule[152]=C:\WINDOWS\system32\dataexchange.dll
LoadedModule[153]=C:\Program Files (x86)\Microsoft Visual Studio\Shared\Visual Studio Tools for Office\x86\VSTOExcelAdaptor.dll
LoadedModule[154]=C:\WINDOWS\assembly\NativeImages_v4.0.30319_32\System.Xml\f71225c38cd3420200689412958881f6\System.Xml.ni.dll
LoadedModule[155]=C:\WINDOWS\assembly\NativeImages_v4.0.30319_32\System.Security\36effc9dd742489d9037858081b86f7d\System.Security.ni.dll
LoadedModule[156]=C:\WINDOWS\SYSTEM32\PROPSYS.dll
LoadedModule[157]=C:\WINDOWS\assembly\NativeImages_v4.0.30319_32\System.Configuration\0c052fccd4c571e447fdc8dd115777d8\System.Configuration.ni.dll
LoadedModule[158]=C:\Program Files (x86)\Microsoft Office\root\Office16\gfx.dll
LoadedModule[159]=C:\WINDOWS\SYSTEM32\srpapi.dll
LoadedModule[160]=C:\Program Files (x86)\Common Files\Microsoft Shared\VBA\VBA7.1\VBE7.DLL
LoadedModule[161]=C:\WINDOWS\SYSTEM32\UIAutomationCore.DLL
LoadedModule[162]=C:\WINDOWS\WinSxS\x86_microsoft.windows.common-controls_6595b64144ccf1df_5.82.17134.648_none_40ff37d245123ba5\COMCTL32.dll
LoadedModule[163]=C:\Program Files (x86)\Common Files\Microsoft Shared\VBA\VBA7.1\VBEUI.DLL
LoadedModule[164]=C:\WINDOWS\System32\WINTRUST.dll
LoadedModule[165]=C:\Program Files (x86)\Common Files\Microsoft Shared\VBA\VBA7.1\1033\VBE7INTL.DLL
LoadedModule[166]=C:\WINDOWS\assembly\NativeImages_v4.0.30319_32\CustomMarshalers\9ae32454132e4f0849753062ec6e2834\CustomMarshalers.ni.dll
LoadedModule[167]=C:\WINDOWS\Microsoft.Net\assembly\GAC_32\CustomMarshalers\v4.0_4.0.0.0__b03f5f7f11d50a3a\CustomMarshalers.dll
LoadedModule[168]=C:\WINDOWS\assembly\NativeImages_v4.0.30319_32\Microsoft.CSharp\50563b84d4b178cb6ace5be511a2f534\Microsoft.CSharp.ni.dll
LoadedModule[169]=C:\WINDOWS\assembly\NativeImages_v4.0.30319_32\System.Dynamic\08f0eb5ea222045a383e2ada47f96532\System.Dynamic.ni.dll
LoadedModule[170]=C:\WINDOWS\assembly\NativeImages_v4.0.30319_32\System.Data\dc4dceb5cb327e684dd93ea13ef9a286\System.Data.ni.dll
LoadedModule[171]=C:\WINDOWS\Microsoft.Net\assembly\GAC_32\System.Data\v4.0_4.0.0.0__b77a5c561934e089\System.Data.dll
LoadedModule[172]=C:\Program Files (x86)\Microsoft Office\root\Office16\OSFSHARED.DLL
LoadedModule[173]=C:\Program Files (x86)\Microsoft Office\root\Office16\MSOHEV.DLL
LoadedModule[174]=C:\Program Files (x86)\Microsoft Office\Root\Office16\MSIPC\msipc.dll
LoadedModule[175]=C:\Program Files (x86)\Microsoft Office\Root\Office16\MSIPC\ipcsecproc.dll
LoadedModule[176]=C:\WINDOWS\SYSTEM32\LINKINFO.dll
LoadedModule[177]=C:\Program Files (x86)\Microsoft Office\root\Office16\MsoAriaNext.dll
LoadedModule[178]=C:\Windows\System32\Windows.Networking.Connectivity.dll
LoadedModule[179]=C:\Windows\System32\Windows.Security.Authentication.OnlineId.dll
LoadedModule[180]=C:\Windows\System32\OneCoreUAPCommonProxyStub.dll
LoadedModule[181]=C:\Program Files (x86)\Microsoft Office\root\Office16\OSF.DLL
LoadedModule[182]=C:\Program Files (x86)\Microsoft Office\root\Office16\OSFUI.DLL
LoadedModule[183]=C:\WINDOWS\SYSTEM32\rometadata.dll
LoadedModule[184]=C:\Program Files (x86)\Common Files\Microsoft Shared\Office16\1033\OSFINTL.DLL
LoadedModule[185]=C:\Windows\System32\ieproxy.dll
LoadedModule[186]=C:\Program Files (x86)\Microsoft Office\Root\Office16\OSFROAMINGPROXY.DLL
LoadedModule[187]=C:\WINDOWS\SYSTEM32\FirewallAPI.dll
LoadedModule[188]=C:\WINDOWS\SYSTEM32\fwbase.dll
LoadedModule[189]=C:\WINDOWS\SYSTEM32\D3D10Warp.dll
LoadedModule[190]=C:\WINDOWS\SYSTEM32\gpapi.dll
State[0].Key=Transport.DoneStage1
State[0].Value=1
ProcessMetadata[0].Key=OfficeProcArch
ProcessMetadata[0].Value=x86
ProcessMetadata[1].Key=InstallType
ProcessMetadata[1].Value=2
ProcessMetadata[2].Key=DeviceClass
ProcessMetadata[3].Key=Channel
ProcessMetadata[3].Value=CC
ProcessMetadata[4].Key=AudienceGroup
ProcessMetadata[4].Value=Production
OsInfo[0].Key=vermaj
OsInfo[0].Value=10
OsInfo[1].Key=vermin
OsInfo[1].Value=0
OsInfo[2].Key=verbld
OsInfo[2].Value=17134
OsInfo[3].Key=ubr
OsInfo[3].Value=648
OsInfo[4].Key=versp
OsInfo[4].Value=0
OsInfo[5].Key=arch
OsInfo[5].Value=9
OsInfo[6].Key=lcid
OsInfo[6].Value=1033
OsInfo[7].Key=geoid
OsInfo[7].Value=77
OsInfo[8].Key=sku
OsInfo[8].Value=101
OsInfo[9].Key=domain
OsInfo[9].Value=0
OsInfo[10].Key=prodsuite
OsInfo[10].Value=768
OsInfo[11].Key=ntprodtype
OsInfo[11].Value=1
OsInfo[12].Key=platid
OsInfo[12].Value=10
OsInfo[13].Key=sr
OsInfo[13].Value=0
OsInfo[14].Key=tmsi
OsInfo[14].Value=423647
OsInfo[15].Key=osinsty
OsInfo[15].Value=1
OsInfo[16].Key=iever
OsInfo[16].Value=11.648.17134.0-11.0.115
OsInfo[17].Key=portos
OsInfo[17].Value=0
OsInfo[18].Key=ram
OsInfo[18].Value=16270
OsInfo[19].Key=svolsz
OsInfo[19].Value=475
OsInfo[20].Key=wimbt
OsInfo[20].Value=0
OsInfo[21].Key=blddt
OsInfo[21].Value=180410
OsInfo[22].Key=bldtm
OsInfo[22].Value=1804
OsInfo[23].Key=bldbrch
OsInfo[23].Value=rs4_release
OsInfo[24].Key=bldchk
OsInfo[24].Value=0
OsInfo[25].Key=wpvermaj
OsInfo[25].Value=0
OsInfo[26].Key=wpvermin
OsInfo[26].Value=0
OsInfo[27].Key=wpbuildmaj
OsInfo[27].Value=0
OsInfo[28].Key=wpbuildmin
OsInfo[28].Value=0
OsInfo[29].Key=osver
OsInfo[29].Value=10.0.17134.648.amd64fre.rs4_release.180410-1804
OsInfo[30].Key=buildflightid
OsInfo[30].Value=0AA5E289-E8CB-4F08-ACEB-971D71933A65.1
OsInfo[31].Key=edition
OsInfo[31].Value=Core
OsInfo[32].Key=ring
OsInfo[33].Key=expid
OsInfo[34].Key=containerid
OsInfo[35].Key=containertype
OsInfo[36].Key=edu
OsInfo[36].Value=0
FriendlyEventName=Stopped working
ConsentKey=APPCRASH
AppName=Microsoft Excel
AppPath=C:\Program Files (x86)\Microsoft Office\root\Office16\EXCEL.EXE
ApplicationIdentity=00000000000000000000000000000000
MetadataHash=1631255944
```

**Comment by dmahugh** [comment]
@JuaneloJuanelo and/or @keyur32, here's another report of a simple action that crashes Excel, with a repro.

**Comment by tutieu** [comment]
This issues is repro'ed.  Tracking bug 3285352.

**Comment by lumine2008** [comment]
This issue was fixed, Fixed Build: 16.0.11624.10000.
